ppr-ui/src/components/registration/RegistrationLengthTrust.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <v-container fluid no-gutters class="white pa-0" v-if="summaryView">
     <v-card flat id="length-trust-summary">
-      <v-row no-gutters class="summary-header pa-2">
+      <v-row class="py-2 renewal-title" v-if="isRenewal">
```

Comment:
do you need no-gutters here? Also, are you getting anything from nesting the `<h2>` within the `v-row / v-col`? Might be able to just have the `<h2>` with the `v-if` and the `class`

Reply:
changed it to just the h2